src/FixerDefinition/VersionSpecificCodeSample.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\FixerDefinition;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ */
+final class VersionSpecificCodeSample implements VersionSpecificCodeSampleInterface
+{
+    /**
+     * @var CodeSampleInterface
+     */
+    private $codeSample;
+
+    /**
+     * @var null|VersionSpecificationInterface
+     */
+    private $versionSpecification;
+
+    /**
+     * @param string                             $code
+     * @param null|array                         $configuration
+     * @param VersionSpecificationInterface|null $versionSpecification
+     */
+    public function __construct(
+        $code,
+        array $configuration = null,
+        VersionSpecificationInterface $versionSpecification = null
```

Comment:
Well, that would just mean that we need to make `$configuration` a required value, right? Or should I modify the order of arguments, and make `$versionSpecification` the second argument and required?